tests/test_statistics.py

```diff
@@ -573,6 +573,51 @@ def test_std_error(self):
             dpnp.std(ia, ddof="1")
 
 
+class TestCorrcoef:
```

Comment:
Could you please add a test per `test_sycl_queue.py` and `test_usm_type.py` scopes?

Reply:
Added